kubemarine/resources/configurations/defaults.yaml

```diff
@@ -717,7 +717,7 @@ rbac:
           annotations: {}
         type: kubernetes.io/service-account-token
 
-  admission: psp
+  admission: pss
```

Comment:
This is incompatible change as Kubemarine suddenly starts to install different admission implementation on old clusters. I think that default admission implementation should be calculated dynamically based on kubernetes version if no explicit implementation is specified. @alexarefev

Reply:
Definitely we need an improvement here